sympy/series/tests/test_residues.py

```diff
@@ -78,4 +78,12 @@ def test_issue_14037():
 
 
 def test_issue_21176():
-    assert residue(x**2*cot(pi*x)/(x**4 + 1), x, -sqrt(2)/2 - sqrt(2)*I/2) == 0
+    f = x**2*cot(pi*x)/(x**4 + 1)
+    assert residue(f, x, -sqrt(2)/2 - sqrt(2)*I/2).simplify() == \
```

Comment:
`simplify` is a fairly expensive method. Something like `together` might suffice. Tests should be made as fast as possible.